docs/README.md

```diff
@@ -206,6 +206,12 @@ By default implementation we mean that whenever you use `product-list` in your s
 
 Therefore, in order to customize the Product List configuration, you can simply use the default implementation in your code and change it as you wish.
 
+| Prop name              | Type   | Description                                                                                                                                   | Default value |
+| ---------------------- | ------ | --------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| `unitPriceDisplay`     | `Enum` | Defines whether the unit price should always be displayed (`always`) or only when the product quantity is bigger than one (`default`) | `default`   |
+| `displayUnitListPrice` | `Enum` | Defines whether the unit list price should be displayed when different from price (`always`) or not (`never`)    | `never`     |
```

Comment:
Is there a reason to display the list price if the price is the same?

Reply:
@klzns No, list price will only be shown when it is different from price

Reply:
So I guess we can remove this prop. What do you think?

Reply:
@klzns this property will show the list price when it is different from the price, it is currently not shown. Maybe the documentation is causing some confusion

Reply:
@klzns I think this prop is important because of the current stores that not implement this HTML element. Think: If the store customers don't have this element and we don't allow them to  able and disable this element, this will broke some layouts.